recipe/meta.yaml

```
@@ -11,32 +11,35 @@ source:
     - patches/Makefile.patch
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
   run_exports:
     - {{ pin_subpackage("lmdb", max_pin="x.x.x") }}
 
 requirements:
   build:
     - make
+    - cmake  # [win]
     - {{ compiler('c') }}
 
 test:
   commands:
     # binaries
-    - mdb_copy -V
-    - mdb_dump -V
-    - mdb_load -V
-    - mdb_stat -V
+    - mdb_copy -V  # [unix]
+    - mdb_dump -V  # [unix]
+    - mdb_load -V  # [unix]
+    - mdb_stat -V  # [unix]
 
     # presence of shared lib:
-    - test -f ${PREFIX}/lib/liblmdb.so
+    - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
+    - if not exist %LIBRARY_BIN%\lmdb.dll exit 1  # [win]
```

`.lib` has a double meaning on windows - it can be a static library or an "import library". The latter is necessary to actually use any symbols from the DLL. This is the `/IMPLIB` stuff that you removed. 
```suggestion
    - test -f ${PREFIX}/lib/liblmdb.so               # [unix]
    - if not exist %LIBRARY_BIN%\lmdb.dll exit 1     # [win]
    - if not exist %LIBRARY_LIB%\lmdb.lib exit 1     # [win]
```

---

It's possible that you'll need `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON` to ensure the import library is created. If that's not enough you'll need to add something like [ABSL_DLL](https://github.com/abseil/abseil-cpp/blob/20230125.3/absl/base/config.h#L743-L760), e.g.:
```
#if defined(LMDB_DLL_EXPORTS)
#define LMDB_DLL __declspec(dllexport)
#elif defined(LMDB_DLL_IMPORTS)
#define LMDB_DLL __declspec(dllimport)
#else
#define LMBD_DLL
#endif // defined(LMBD_DLL_EXPORTS)
```

plus some CMake magic:
```
  if(BUILD_SHARED_LIBS)
    set_target_properties(lmdb PROPERTIES DEFINE_SYMBOL "LMDB_DLL_EXPORTS")
    target_compile_definitions(gpr INTERFACE LMDB_DLL_IMPORTS)
  endif()
```

and then mark every required class/symbol with `LMBD_DLL`.